src/js/outreach/views/dialog.scss

```diff
@@ -20,6 +20,7 @@
   margin: 0 auto;
   max-width: 520px;
   padding: 48px 32px;
+  text-align: left;
```

Comment:
I need to go through this file and remove any un-used code. And I could probably consolidate some styles that are identical between the `verify.scss` and `opt-in.scss` files.

Reply:
Also need to update the error views ([example](https://github.com/RoundingWell/care-ops-frontend/assets/35355575/4a081a7e-7adb-4a8d-91d3-4fce3edf871a)) to use the new icons and spacing from the design docs.